content/en/serverless/azure_app_services.md

```diff
@@ -41,11 +42,16 @@ The Datadog extension for Azure App Services provides additional monitoring capa
 
 1. If you haven't already, set up the [Microsoft Azure integration][8] first.
 
-2. The extension only supports Azure App Service Web Apps. **Function Apps are not supported**.
+2. The extension only supports:
 
-<div class="alert alert-warning">To be notified when a private beta is available for Function Apps or additional runtimes, <a href="https://forms.gle/n4nQcxEyLqDBMCDA7">sign up here</a>.</div>
+    - Azure App Service Web Apps
+    - Function Apps hosted on Premium or Standard App Service Plans (beta for extension v1.3.4+). **Note**: Function Apps on consumption plans are not supported.
```

Comment:
**Notes**:
- Function Apps on consumption plans are not supported.
- There are no billing implications for tracing Function apps during the beta period.